Source/Python/WarpX.cpp

```diff
@@ -143,6 +143,10 @@ void init_WarpX (py::module& m)
             [](WarpX const & wx, int lev){ return wx.gett_new(lev); },
             py::arg("lev")
         )
+        .def("getdt",
+            [](WarpX const & wx, int lev){ return wx.getdt(lev); },
```

Comment:
Does this work as well? :)
```suggestion
        .def("getdt", &WarpX::getdt,
```